src/utils/general/mergeReactRefs.ts

```diff
@@ -0,0 +1,13 @@
+export function mergeRefs<T = any>(
+  refs: Array<React.MutableRefObject<T> | React.LegacyRef<T>>
```

Comment:
Curious: why `LegacyRef`?

Reply:
I took the code from the lib https://www.npmjs.com/package/react-merge-refs

I suppose it's for supporting class components ref